lib/inc/scrutiny_tools.h

```diff
@@ -47,6 +48,25 @@ namespace scrutiny
             }
             return n;
         }
+
+        class MulBerry32PRNG
```

Comment:
potential breach of licence. code copy/pasta-ed from the internet without attribution/reference